src/main.c

```diff
@@ -1426,9 +1426,43 @@ int main(int argc, char * argv [])
     goto main_exit;
   }
 
+#ifdef HAVE_LIBELF
+  rc = 0;
+  for (ln=lfirst(updates); ln; ln=lnext(ln)) {
+    upd = ldata(ln);
+    if (avr_memtype_is_flash_type(upd->memtype) && (upd->op == DEVICE_WRITE) &&
+        upd->elf_all_write) {
+      rc++;
+    }
+  }
+  if (rc > 1) {
+    pmsg_error("option -U file.elf can be used only once!!!\n");
```

Comment:
We should try to make this error a little less angry. How about
`pmsg_error("option -U file.elf can only be used once.\n");` ?

Reply:
I did.